_authentication.md

```diff
@@ -244,4 +244,6 @@ curl https://api.uphold.com/v0/me \
 
 You can use Basic Authentication by providing your username or email and password combination.
 
-If OTP (One Time Password, also known as Two-Factor Authentication) is required, then you will get an HTTP 401 (Unauthorized) response, along with the HTTP header `OTP-Token: Required`. Execute the command above again, this time passing your OTP verification code as a header, like so: `OTP-Token: <OTP-Token>`.
+If OTP (One-Time Password, also known as Two-Factor Authentication) is required, then you will get an HTTP 401 (Unauthorized) response, along with the HTTP header `OTP-Token: Required` or `OTP-Method-Id: Required`.
+For the former case, execute the command above again, this time passing your OTP verification code and phone id as a headers, like so: `OTP-Phone-Id: <OTP-Phone-Id>` (deprecated) and `OTP-Token: <OTP-Token>`.
+For the latter case, execute the command above again, this time passing your OTP verification code and method id as a headers, like so: `OTP-Method-Id: <OTP-Method-Id>` and `OTP-Token: <OTP-Token>`.
```

Comment:
Nit-pick: this starts by referencing token + id, but when showing the headers it's id + token. Keep the same order, for better readability?